File: FutronicSDK/Examples/Net/Vs2003/WorkedEx/EnrollmentName.cs
using System;
using System.Drawing;
using System.Collections;
using System.ComponentModel;
using System.Windows.Forms;

namespace Futronic.SDK.WorkedEx
{
	/// <summary>
	/// Summary description for EnrollmentName.
	/// </summary>
	public class EnrollmentName : System.Windows.Forms.Form
	{
		private System.Windows.Forms.Label label1;
		private System.Windows.Forms.TextBox txtUserName;
		private System.Windows.Forms.Button btnOK;
		private System.Windows.Forms.Button btnCancel;
		/// <summary>
		/// Required designer variable.
		/// </summary>
		private System.ComponentModel.Container components = null;

		public EnrollmentName()
		{
			//
			// Required for Windows Form Designer support
			//
			InitializeComponent();

		}

		public String UserName
		{
			get
			{
				return txtUserName.Text;
			}
		}

		/// <summary>
		/// Clean up any resources being used.
		/// </summary>
		protected override void Dispose( bool disposing )
		{
			if( disposing )
			{
				if(components != null)
				{
					components.Dispose();
				}
			}
			base.Dispose( disposing );
		}

		#region Windows Form Designer generated code
		/// <summary>
		/// Required method for Designer support - do not modify
		/// the contents of this method with the code editor.
		/// </summary>
		private void InitializeComponent()
		{
			this.label1 = new System.Windows.Forms.Label();
			this.txtUserName = new System.Windows.Forms.TextBox();
			this.btnOK = new System.Windows.Forms.Button();
			this.btnCancel = new System.Windows.Forms.Button();
			this.SuspendLayout();
			// 
			// label1
			// 
			this.label1.Location = new System.Drawing.Point(8, 40);
			this.label1.Name = "label1";
			this.label1.Size = new System.Drawing.Size(64, 16);
			this.label1.TabIndex = 0;
			this.label1.Text = "User name";
			// 
			// txtUserName
			// 
			this.txtUserName.Location = new System.Drawing.Point(72, 38);
			this.txtUserName.Name = "txtUserName";
			this.txtUserName.Size = new System.Drawing.Size(232, 20);
			this.txtUserName.TabIndex = 1;
			this.txtUserName.Text = "";
			// 
			// btnOK
			// 
			this.btnOK.DialogResult = System.Windows.Forms.DialogResult.OK;
			this.btnOK.Location = new System.Drawing.Point(336, 16);
			this.btnOK.Name = "btnOK";
			this.btnOK.TabIndex = 2;
			this.btnOK.Text = "Ok";
			// 
			// btnCancel
			// 
			this.btnCancel.DialogResult = System.Windows.Forms.DialogResult.Cancel;
			this.btnCancel.Location = new System.Drawing.Point(336, 48);
			this.btnCancel.Name = "btnCancel";
			this.btnCancel.TabIndex = 3;
			this.btnCancel.Text = "Cancel";
			// 
			// EnrollmentName
			// 
			this.AutoScaleBaseSize = new System.Drawing.Size(5, 13);
			this.ClientSize = new System.Drawing.Size(422, 96);
			this.ControlBox = false;
			this.Controls.Add(this.btnCancel);
			this.Controls.Add(this.btnOK);
			this.Controls.Add(this.txtUserName);
			this.Controls.Add(this.label1);
			this.Name = "EnrollmentName";
			this.StartPosition = System.Windows.Forms.FormStartPosition.CenterParent;
			this.Text = "User name";
			this.ResumeLayout(false);

		}
		#endregion
	}
}

File: FutronicSDK/Examples/Net/Vs2003/WorkedEx.VB/EnrollmentName.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Drawing
Imports System.Collections
Imports System.ComponentModel
Imports System.Windows.Forms

Namespace Futronic.SDK.WorkedEx
	''' <summary>
	''' Summary description for EnrollmentName.
	''' </summary>
	Public Class EnrollmentName
		Inherits System.Windows.Forms.Form
		Private label1 As System.Windows.Forms.Label
		Private txtUserName As System.Windows.Forms.TextBox
		Private btnOK As System.Windows.Forms.Button
		Private btnCancel As System.Windows.Forms.Button
		''' <summary>
		''' Required designer variable.
		''' </summary>
		Private components As System.ComponentModel.Container = Nothing

		Public Sub New()
			'
			' Required for Windows Form Designer support
			'
			InitializeComponent()

		End Sub

		Public ReadOnly Property UserName() As String
			Get
				Return txtUserName.Text
			End Get
		End Property

		''' <summary>
		''' Clean up any resources being used.
		''' </summary>
		Protected Overrides Overloads Sub Dispose(ByVal disposing As Boolean)
			If disposing Then
				If Not components Is Nothing Then
					components.Dispose()
				End If
			End If
			MyBase.Dispose(disposing)
		End Sub

		#Region "Windows Form Designer generated code"
		''' <summary>
		''' Required method for Designer support - do not modify
		''' the contents of this method with the code editor.
		''' </summary>
		Private Sub InitializeComponent()
			Me.label1 = New System.Windows.Forms.Label()
			Me.txtUserName = New System.Windows.Forms.TextBox()
			Me.btnOK = New System.Windows.Forms.Button()
			Me.btnCancel = New System.Windows.Forms.Button()
			Me.SuspendLayout()
			' 
			' label1
			' 
			Me.label1.Location = New System.Drawing.Point(8, 40)
			Me.label1.Name = "label1"
			Me.label1.Size = New System.Drawing.Size(64, 16)
			Me.label1.TabIndex = 0
			Me.label1.Text = "User name"
			' 
			' txtUserName
			' 
			Me.txtUserName.Location = New System.Drawing.Point(72, 38)
			Me.txtUserName.Name = "txtUserName"
			Me.txtUserName.Size = New System.Drawing.Size(232, 20)
			Me.txtUserName.TabIndex = 1
			Me.txtUserName.Text = ""
			' 
			' btnOK
			' 
			Me.btnOK.DialogResult = System.Windows.Forms.DialogResult.OK
			Me.btnOK.Location = New System.Drawing.Point(336, 16)
			Me.btnOK.Name = "btnOK"
			Me.btnOK.TabIndex = 2
			Me.btnOK.Text = "Ok"
			' 
			' btnCancel
			' 
			Me.btnCancel.DialogResult = System.Windows.Forms.DialogResult.Cancel
			Me.btnCancel.Location = New System.Drawing.Point(336, 48)
			Me.btnCancel.Name = "btnCancel"
			Me.btnCancel.TabIndex = 3
			Me.btnCancel.Text = "Cancel"
			' 
			' EnrollmentName
			' 
			Me.AutoScaleBaseSize = New System.Drawing.Size(5, 13)
			Me.ClientSize = New System.Drawing.Size(422, 96)
			Me.ControlBox = False
			Me.Controls.Add(Me.btnCancel)
			Me.Controls.Add(Me.btnOK)
			Me.Controls.Add(Me.txtUserName)
			Me.Controls.Add(Me.label1)
			Me.Name = "EnrollmentName"
			Me.StartPosition = System.Windows.Forms.FormStartPosition.CenterParent
			Me.Text = "User name"
			Me.ResumeLayout(False)

		End Sub
		#End Region
	End Class
End Namespace
